### DISCUSSION TITLE
Initial velocity and Energy consumption of the vehicle

### DISCUSSION THREAD

**Discussion by woojmn** [DLR-VSDC/IEEE-MVC-2023#11] [Q&A] [answered]
Hello, I would like to discuss some questions.

We have two questions while running the Simulink model.

1. We think the initial velocity can be checked in the 'logVel' of the 'chassisBus'. The initial velocity is 1, not zero. Is that correct and Is there any problem for design the EMA?

2. We confirmed that the energy consumption cost(J_energy) can be negative. When the alpha_FC=1, alpha_TV=0, alpha_AD=1 and alpha_v = 0, although the velocity does not follow the reference velocity, we think that the energy consumption is always positive because the power from fuel cell cannot be less than battery power without regenerative brake. However, when the alpha_FC=1, alpha_TV=0, alpha_AD=1 and alpha_v = 0, the velocity is almost zero, while the energy consumption cost is negative. We wonder if this is possible and why it happens.

**Comment by JBRDLR** [comment]
Dear @woojmn is this behavior in Beta 2 (yesterdays latest update ) and please provide here some plots/data? @imeb can you please answer these questions?

**Reply by woojmn** [reply]
Yes, there are same problem in Beta 2.
The velocity and J_energy are shown in below image.

![image](https://user-images.githubusercontent.com/95162423/210737456-62562596-cfaa-437f-b567-1e63ede0cbf5.png)
![image](https://user-images.githubusercontent.com/95162423/210737465-a4901acd-8228-46f6-b937-65df30289ae4.png)

**Reply by JBRDLR** [reply]
Can you please give more information on the manouver, parameters, etc. we are looking here. Otherwise, it is hard to reproduce and analyze the issue.
Regarding the initial velocity, that might bei an initialization problem, but should not concern the overall simulation. @imeb you should set V_init0 to the first value of the particular drive cycle in the main script. As far as they  differ from each other

**Comment by JBRDLR** [comment, marked as answer]
@woojmna new beta 3 will be available by end of week. It will fix these issues.
Best Jonathan
